tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -29,6 +29,11 @@ type TestCase struct {
 	Jwt           string
 }
 
+const (
+	gRPCDenyMsg = "rpc error: code = PermissionDenied desc = RBAC: access denied"
+	tcpDenyMsg  = "EOF"
```

Comment:
These constants are only used in one place and not likely to be reused anywhere else, it will be more readable to just hard-code them in the place of calling.

Reply:
Sounds good